src/components/qualityOptions.js

```diff
@@ -53,7 +53,7 @@ export function getVideoQualityOptions(options) {
 
     if (maxStreamingBitrate) {
         let selectedIndex = -1;
-        for (let i = 0, length = qualityOptions.length; i < length; i++) {
+        for (let i = 1, length = qualityOptions.length; i < length; i++) {
             const option = qualityOptions[i];
 
             if (selectedIndex === -1 && option.bitrate <= maxStreamingBitrate) {
```

Comment:
Given that the bitrates are sorted in descending order, we can stop the loop as soon as `option.bitrate <= maxStreamingBitrate` and remove `selectedIndex === -1`. The same for audio.